src/libraries/System.DirectoryServices.AccountManagement/src/System/DirectoryServices/AccountManagement/AdvancedFilters.cs

```diff
@@ -25,68 +25,50 @@ protected internal AdvancedFilters(Principal p)
 
         public void LastBadPasswordAttempt(DateTime lastAttempt, MatchType match)
         {
-            if (lastAttempt == null)
+            if (null == _badPasswordAttemptVal)
```

Comment:
I thought we generally avoided Yoda- style conditionals? They don't gain anything in C# (except for bools)

Reply:
I didn't add this code.  It's the way it's currently written.  I'm just deleting code.